project/app/components/MemberInfo.js

```diff
@@ -5,7 +5,7 @@ import classNames from 'classnames';
 import { SpacesLogo } from '@availity/spaces';
 import { useQuery } from 'react-query';
 
-import { useAppContext } from '@/context';
+import { useAppContext } from '../context';
```

Comment:
I don't think we should change these imports. We can fix it by adding `tsconfig-paths-webpack-plugin` to workflow

Reply:
I added this: `plugins: [new TsconfigPathsPlugin()]`, to the `webpack.config.js` file under `config.resolve`.

Reply:
Sounds good, I'll update. Thanks

Reply:
this has been fixed.